src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,11 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        StringBuilder result = new StringBuilder();
+        do {
+            result.insert(0,value % 2);
```

Comment:
please, use `append` method instead of `insert`. it's easier to read and understand what it does